apps/website/src/data/network.ts

```diff
@@ -31,6 +31,10 @@ type NetworkItemCamera = NetworkItemCore & {
   type: "camera" | "microphone";
 };
 
+type NetworkItemSpeaker = NetworkItemCore & {
+  type: "speaker";
+};
```

Comment:
```suggestion
  type: "camera" | "microphone" | "speaker";
};
```
I think we can just extend the existing camera type, rather than defining a dedicated speaker type, so they're grouped by color in the UI.